utils/table-helper.js

```diff
@@ -0,0 +1,11 @@
+const _ = require('lodash');
+
+// NOTICE: Look for the id column in both fields and primary keys
+function hasIdColumn(fields, primaryKeys) {
+  return fields.some(field => field.name === 'id' || field.nameColumn === 'id') ||
+        _.includes(primaryKeys, 'id');
```

Comment:
[Code Style] Maybe we should add ESLint, the indentation is pretty bad here.
To replace by:

```javascript
return fields.some(field => field.name === 'id' || field.nameColumn === 'id')
  || _.includes(primaryKeys, 'id');
```